src/webgpu/api/validation/attachment_compatibility.spec.ts

```diff
@@ -12,11 +12,13 @@ import {
   kRegularTextureFormats,
   kSizedDepthStencilFormats,
   kUnsizedDepthStencilFormats,
-  RegularTextureFormat,
 } from '../../capability_info.js';
 
 import { ValidationTest, CommandBufferMaker } from './validation_test.js';
 
+// TODO: Update with all possible sample counts when defined
+const kSampleCounts = [1, 4] as const;
```

Comment:
nit: Could you put this definition in capability_info.ts? We probably already have other tests that hardcode 1 and 4, but putting it there helps keep track of it for optional capabilities, like sample count 8, in the future. When you do that, please also add a TODO to find other tests for sample counts and use kSampleCounts in them.

Reply:
Done